crates/rolldown/src/module_finalizers/hmr.rs

```diff
@@ -108,6 +108,10 @@ impl<'ast> ScopeHoistingFinalizer<'_, 'ast> {
   }
 
   pub fn rewrite_hot_accept_call_deps(&self, call_expr: &mut ast::CallExpression<'ast>) {
+    if !self.ctx.options.is_hmr_enabled() {
```

Comment:
Is this the opppisite to this PR title?

Reply:
https://github.com/rolldown/rolldown/blob/72bdeb3a45af3e30f7535efd4be9edfcfcb12066/crates/rolldown/src/ast_scanner/hmr.rs?plain=1#L12-L14

Reply:
> Is this the opppisite to this PR title?

I think they just represent the same meaning, since this is early return, align with related code in `ast_scanner`